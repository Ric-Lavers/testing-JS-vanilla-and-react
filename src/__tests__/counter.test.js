import React from 'react';
import renderer from 'react-test-renderer';
import Counter from '../Counter';

describe('A counter', () => {
  it('should display the count', () => {
    const counter = renderer.create(<Counter ledgend="Count" count={34}/>)
    expect(counter.toJSON()).toMatchSnapshot();
  });
});

// `npx jest -u` use if the change is a legitimate change and the html snapshop needs to be updated.
