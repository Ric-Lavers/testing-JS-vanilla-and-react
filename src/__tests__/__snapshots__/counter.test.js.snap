// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A counter should display the count 1`] = `
<p
  className="mb2"
>
  Word Count: 
  34
</p>
`;
